### PR TITLE
Migrating to Androidx

### PR DESCRIPTION
**Pull request by rurimo**
Migrating to Androidx

### CODE REVIEW

File: AudioProcessor/build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ apply from: "$rootProject.rootDir/gradle/bintray.gradle"
 
 android {
     compileSdkVersion rootProject.ext.androidCompileSdkVersion
-    buildToolsVersion rootProject.ext.androidBuildToolsVersion
+    buildToolsVersion '28.0.3'
```
Comment (Piasy) [inline AudioProcessor/build.gradle]: Why not change `androidBuildToolsVersion` in root build.gradle?

File: app/build.gradle [hunk 2]
```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion rootProject.ext.androidCompileSdkVersion
-    buildToolsVersion rootProject.ext.androidBuildToolsVersion
+    buildToolsVersion '28.0.3'
```
Comment (Piasy) [inline app/build.gradle]: Why not change `androidBuildToolsVersion` in root build.gradle?

File: app/build.gradle [hunk 3]
```diff
@@ -26,17 +26,17 @@ android {
 }
 
 dependencies {
-    implementation "com.android.support:appcompat-v7:${rootProject.ext.androidSupportSdkVersion}"
+    implementation 'androidx.appcompat:appcompat:1.0.2'
```
Comment (Piasy) [inline app/build.gradle:29]: Why not add an `androidXVersion` in root build.gradle?

File: app/src/main/java/com/github/piasy/rxandroidaudio/example/FileActivity.java [hunk 4]
```diff
@@ -94,13 +94,13 @@ protected void onCreate(Bundle savedInstanceState) {
...
 
         mIvVoiceIndicators = new ArrayList<>();
-        mIvVoiceIndicators.add(ButterKnife.findById(this, R.id.mIvVoiceIndicator1));
-        mIvVoiceIndicators.add(ButterKnife.findById(this, R.id.mIvVoiceIndicator2));
-        mIvVoiceIndicators.add(ButterKnife.findById(this, R.id.mIvVoiceIndicator3));
-        mIvVoiceIndicators.add(ButterKnife.findById(this, R.id.mIvVoiceIndicator4));
-        mIvVoiceIndicators.add(ButterKnife.findById(this, R.id.mIvVoiceIndicator5));
-        mIvVoiceIndicators.add(ButterKnife.findById(this, R.id.mIvVoiceIndicator6));
-        mIvVoiceIndicators.add(ButterKnife.findById(this, R.id.mIvVoiceIndicator7));
+        mIvVoiceIndicators.add((ImageView) findViewById(R.id.mIvVoiceIndicator1));
```
Comment (Piasy) [inline app/src/main/java/com/github/piasy/rxandroidaudio/example/FileActivity.java:106]: Do we still need the type cast?

File: rxandroidaudio/src/main/java/com/github/piasy/rxandroidaudio/PlayConfig.java [hunk 5]
```diff
@@ -157,9 +159,6 @@ boolean needPrepare() {
         @FloatRange(from = 0.0F, to = 1.0F)
         float mRightVolume = 1.0F;
 
-        /**
```
Comment (Piasy) [inline rxandroidaudio/src/main/java/com/github/piasy/rxandroidaudio/PlayConfig.java]: Why do you delete this comment?

**Review by Piasy — CHANGES_REQUESTED** [review]

**Comment by Piasy** [comment]
Thanks for your contribution! I have some comments on your changes, please address them so that I can merge it :)

**Comment by rurimo** [comment]
@Piasy 
Thank you for your PR review.
I refactored them by your comments and some example codes. :)

**Comment by Piasy** [comment]
v1.6.0 is released, check it out :)

**Pull request merged**